cmd/snap/cmd_known.go

```diff
@@ -31,11 +31,25 @@ import (
 	"github.com/jessevdk/go-flags"
 )
 
+type assertTypeName string
+
+func (n assertTypeName) Complete(match string) []flags.Completion {
+	names := asserts.TypeNames()
```

Comment:
maybe a comment here about why asserts.TypeNames() is used instead of client.TypeNames()? if we ever make snap talk to a remote snapd or if snap and snapd are different source versions then things might get a bit confusing here.